Docs: fix webhook notification docs in workflow (#223)
oam-dev/kubevela.io: docs/end-user/workflow/webhook-notification.md

---
title:  Webhook Notification
---

> If we want to be notified before and after we deploy an application, KubeVela provides the ability to integrate with webhook notifications, allowing users to send notifications to DingTalk or Slack in their workflow.

Reviewer: ```suggestion
If we want to be notified before or after deploying an application, KubeVela provides integration with notification webhooks, allowing users to send notifications to DingTalk or Slack.
```
